core/templates/dev/head/css/oppia.css

```diff
@@ -1055,7 +1055,7 @@ textarea {
   background: linear-gradient(to right, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
   /*The display is set in Library.js*/
   display: none;
-  height: 240px;
+  height: 259px;
```

Comment:
I think 258px. I seem to remember a recent PR by @souravbadami where the heights of the cards were increased by 18px; he probably didn't notice this one.

Edit: found it! https://github.com/oppia/oppia/pull/1873/files